src/main/java/Ui.java

```diff
@@ -0,0 +1,60 @@
+import Exceptions.DukeException;
+
+/**
+ * Represents the User Interface of the Duke App.
+ * Each <Code>Ui</Code> method displays a message to the user.
+ */
+public class Ui {
+
+    /**
+     * Shows the welcome message when the user first opens the app.
+     * @return The welcome message.
+     */
+    public String showWelcome() {
+        String logo = "I'm Chewy,\n" +
```

Comment:
Perhaps the logo string does not need a "+" operator, it can just be condensed into 1 string.